src/app/worker/base.worker.ts

```
@@ -1,98 +1,96 @@
-        const failedTask = await this.taskService.getTaskById(taskId);
-        if (failedTask) {
-          const coordinator = this.coordinatorFactory.getCoordinator(
-            failedTask.type,
-          );
-          await coordinator.handleTaskFailure(taskId, error);
-        }
-      } catch (coordinatorError) {
-        this.logger.warn(
-          `Coordinator failure handling failed for task ${taskId}: ${coordinatorError.message}`,
-        );
+  protected async processTask(taskId: string): Promise<void> {
```

ESLint

[error] 'taskId' is defined but never used.

(@typescript-eslint/no-unused-vars)